src/pages/style.module.css

```diff
@@ -0,0 +1,22 @@
+.blogTitle {
+  color: black;
+  transition: color 0.2s ease;
+  font-family: "LifeCraft";
+  font-weight: 200;
+}
+
+.blogPostPreview {
+  display: flex;
+  flex-direction: column;
+  border: 2px solid rgba(122, 125, 129, 0.2);
+  border-radius: 8px;
+  padding: 12px;
+  background-image: radial-gradient(white 54%, #d6d6d6 85%);
+  margin: 8px;
+  transition: border 0.2s ease;
+  cursor: pointer;
+}
+
+.blogPostPreview:hover {
+  border: 2px solid rgba(122, 125, 129, 1);
+}
```

Comment:
converted the blog list items to have full element styling and moved the styling to the style sheet in the pages directory